src/entries.ts

```diff
@@ -0,0 +1,2 @@
+export const entries = (obj: any): Array<any[]> => 
```

Comment:
same, ReadonlyArray<T> not Array<T> or T[]

Reply:
Done